config/secrets.yml

```diff
@@ -16,6 +16,16 @@ default: &default
   aws_access_key_id: <%= ENV['AWS_ACCESS_KEY_ID'] %>
   aws_secret_access_key: <%= ENV['AWS_SECRET_ACCESS_KEY'] %>
   aws_bucket: <%= ENV['AWS_BUCKET'] %>
+  enable_persona: <%= ENV['ENABLE_PERSONA'] || true %>
```

Comment:
Don't think we want the secrets file to have opinions about defaults for the ENV values, unless I'm missing something here?